docs/contributing/development_environment.md

```diff
@@ -0,0 +1,232 @@
+Development environment
+=======================
+
+The Adoption development environment utilizes
+[install_yamls](https://github.com/openstack-k8s-operators/install_yamls)
+for CRC VM creation and for creation of the VM that hosts the original
+OpenStack (currently in Standalone configuration).
+
+## Environment prep
+
+Get install_yamls:
+
+```
+git clone https://github.com/openstack-k8s-operators/install_yamls.git
+```
+
+Install tools for operator development:
+
+```
+cd install_yamls/devsetup
+make download_tools
+```
+
+If you want a less intrusive alternative (Go from RPM rather than
+upstream etc.) that allows for basic testing, make sure to at least do
+the following:
+
+```
+sudo dnf -y install \
+    git \
+    golang \
+    guestfs-tools \
+    dbus-x11 \
+    libvirt \
+    make
+
+go env -w GOPROXY="https://proxy.golang.org,direct"
+GO111MODULE=on go install sigs.k8s.io/kustomize/kustomize/v5@latest
+```
+
+## Deployment of CRC with network isolation
+
+```
+cd install_yamls/devsetup
+PULL_SECRET=$HOME/pull-secret.txt CPUS=12 MEMORY=40000 DISK=100 make crc
+
+eval $(crc oc-env)
+oc login -u kubeadmin -p 12345678 https://api.crc.testing:6443
+
+make crc_attach_default_interface
+
+cd ..  # back to install_yamls
+make crc_storage
+make input
+make openstack
+```
+
+## Deployment of Ceph
+
+```
+HOSTNETWORK=false NETWORKS_ANNOTATION=\'[\{\"name\":\"storage\",\"namespace\":\"openstack\"\}]\' MON_IP=172.18.0.30 make ceph TIMEOUT=90
```

Comment:
Makes sense, i'll remove this.

I put this in because i was thinking it would be the easiest path to testing OpenStack adoption with Ceph backends, with Ceph being treated as external. But if we can achieve a more realistic environment with Ceph than that's better.

Thanks for the review!